.circleci/config.yml

```diff
@@ -9,20 +9,42 @@ jobs:
     #   only:
     #     - master
         
-    docker:
-      # specify the version you desire here
-      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
-      - image: circleci/python:3.6.1
+    # docker:
+    #   # specify the version you desire here
+    #   # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+    #   - image: circleci/python:3.6.1
       
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
+    #   # Specify service dependencies here if necessary
+    #   # CircleCI maintains a library of pre-built images
+    #   # documented at https://circleci.com/docs/2.0/circleci-images/
+    #   # - image: circleci/postgres:9.4
```

Comment:
Do you still need these comment sections?

Reply:
resolved in https://github.com/olgaboiar/ttt_python/pull/8/commits/3eb32790195d129740900d935714a1938a88b035